src/Deprecated/Engine/Engine/BuildTask.cs

```diff
@@ -365,11 +365,10 @@ string attributeName
             error.VerifyThrowInvalidOperation(this.taskElement != null,
                 "CannotUseParameters");
 
-            string attributeValue;
-
+            
             // If this is a persisted Task, grab the attribute directly from the
             // task element.
-            attributeValue = taskElement.GetAttribute(attributeName);
+            string attributeValue = taskElement.GetAttribute(attributeName);
 
             return attributeValue ?? String.Empty;
```

Comment:
```suggestion
            return taskElement.GetAttribute(attributeName) ?? String.Empty;
```
This type of change matches with other changes made earlier.

Which raises a good question, do we want to make changes like this? It does make things _slightly_ harder to debug.

Reply:
I definitely vote yes. _Slightly_ harder to debug, maybe, but cleaner and easier to read, definitely.